ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -51,51 +59,41 @@ class TooltipView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 
-    /// Returns the tooltip size
-    static func sizeThatFits(_ size: CGSize,
-                             message: String,
-                             title: String? = nil,
-                             isAccessibilityContentSize: Bool,
-                             arrowDirection: Tooltip.ArrowDirection,
-                             tokenSet: TooltipTokenSet) -> CGSize {
-        let paddingVertical = (title != nil) ? TooltipTokenSet.paddingVerticalWithTitle : TooltipTokenSet.paddingVerticalWithoutTitle
-        let totalPaddingHorizontal = 2 * TooltipTokenSet.paddingHorizontal
-        let arrowHeight = tokenSet[.arrowHeight].float
-        var textBoundingSize = size
-
-        textBoundingSize.width -= totalPaddingHorizontal
-        if !arrowDirection.isVertical {
-            textBoundingSize.width -= arrowHeight
-        }
-
-        let messageLabelFittingSize = labelSizeThatFits(textBoundingSize,
-                                                        text: message,
-                                                        isAccessibilityContentSize: isAccessibilityContentSize,
-                                                        tokenSet: tokenSet,
-                                                        isMessage: true)
-        var width = messageLabelFittingSize.width
-        var height = messageLabelFittingSize.height
-
-        if let title = title {
-            let titleLabelFittingSize = labelSizeThatFits(textBoundingSize,
-                                                          text: title,
-                                                          isAccessibilityContentSize: isAccessibilityContentSize,
-                                                          tokenSet: tokenSet,
-                                                          isMessage: false)
-            width = max(width, titleLabelFittingSize.width)
-            height += titleLabelFittingSize.height
-        }
-
+    func updateTooltipSizeAndOrigin() {
+        updateArrowDirectionAndTooltipRect(for: message, title: titleMessage, tokenSet: tokenSet)
+        self.frame.size = tooltipRect.size
```

Comment:
nit: is it currently ambiguous that you need to call self.frame and self.bounds instead of just frame and bound?